app/lib/screens/OnboardingScreens/Signup.dart

```diff
@@ -18,225 +22,220 @@ class SignupScreen extends StatefulWidget {
+                },
+                child: Form(
+                  key: _formKey,
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.center,
+                    children: [
+                      SizedBox(
+                        height: 80,
+                      ),
+                      SizedBox(
+                        height: 50,
+                        width: 50,
+                        child: SvgPicture.asset('assets/images/logo.svg'),
+                      ),
+                      SizedBox(
+                        height: 40,
+                      ),
+                      Text(
+                        'Lets get Started',
+                        style: TextStyle(
+                          fontSize: 25,
+                          color: Colors.white,
+                          fontWeight: FontWeight.w700,
                         ),
                       ),
-                    ),
-                    SizedBox(
-                      width: 20,
-                    ),
-                    Expanded(
-                      child: Container(
-                        height: 60,
-                        decoration: BoxDecoration(
-                          color: AppCommonTheme.textFieldColor,
-                          borderRadius: BorderRadius.circular(4),
+                      SizedBox(
+                        height: 10,
+                      ),
+                      Text(
+                        'Create an account to explore',
+                        style: TextStyle(
+                          fontSize: 16,
+                          color: Colors.white,
+                          fontWeight: FontWeight.w300,
                         ),
-                        child: TextFormField(
-                          controller: lastNameController,
-                          decoration: InputDecoration(
-                            contentPadding: EdgeInsets.only(
-                              left: 10.0,
-                              top: 12,
-                              right: 10,
-                            ),
-                            border: InputBorder.none,
-                            hintText:
-                                'Last name', // pass the hint text parameter here
-                            hintStyle:
-                                TextStyle(color: AuthTheme.hintTextColor),
+                      ),
+                      SizedBox(
+                        height: 20,
+                      ),
+                      signUpOnboardingTextField(
+                        'Name',
+                        nameController,
+                        'Please enter Your Name',
+                        SignUpOnboardingTextFieldEnum.name,
+                      ),
+                      signUpOnboardingTextField(
+                        'user Name',
+                        userNameController,
+                        'Please enter User Name',
```

Comment:
```suggestion
                        'Please enter Username',
```

Reply:
This gives missing colon error which also needs to be handled automatically as '@'.